graphql/host_resolver.go

```diff
@@ -34,6 +35,32 @@ func (r *hostResolver) Elapsed(ctx context.Context, obj *restModel.APIHost) (*ti
 	return obj.RunningTask.StartTime, nil
 }
 
+// Events is the resolver for the events field.
+func (r *hostResolver) Events(ctx context.Context, obj *restModel.APIHost, limit *int, page *int, sortDir *SortDirection) (*HostEvents, error) {
+	sortAsc := false
+	if sortDir != nil {
+		sortAsc = *sortDir == SortDirectionAsc
+	}
+	events, count, err := event.MostRecentPaginatedHostEvents(utility.FromStringPtr(obj.Id), utility.FromStringPtr(obj.Tag), *limit, *page, sortAsc)
```

Comment:
should we use `utility.FromIntPtr` here?
```suggestion
	events, count, err := event.MostRecentPaginatedHostEvents(utility.FromStringPtr(obj.Id), utility.FromStringPtr(obj.Tag), utility.FromIntPtr(limit), utility.FromIntPtr(page), sortAsc)
```